math_compat.msl

```diff
@@ -1,6 +1,10 @@
 #ifndef METAL_COMPAT_FUNCS
 #define METAL_COMPAT_FUNCS
 
+inline float radians(float angle)
```

Comment:
It's ok if we break this one into `/math/radians.msl` and the rest into `/math/lessThan.msl` and `/math/lessThanEqual.msl` ?

Reply:
Part of my thinking is theres a set of stupid missing features metal doesnt have that is out of the box in other languages? I dunno!

Reply:
the `/math/` folder is ideal for that there is a lot of other polyfill functions in there, some important like this one other just for QOL like `frac/fract`, `lerp/mix`, `saturate`, etc. Other languages like for CUDA they implement all of them just because there is no native version

Reply:
Sure! we can do that. Do you think it makes sense to have a compat folder? My concern is it means that a ton of the existing ported metal shaders will need 1 by 1 different set of includes. 

I like the compat one shot include, but understand it may be different from the rest of the project. 

LMK.